$NetBSD: patch-Makefile.inc2.in,v 1.1 2015/06/05 13:43:20 gdt Exp $

Prepending `depend' and `all' during the install stage will cause at
least two ambiguous errors.  Remove the whole dependency because it
doesn't seem to have much use anyway.

--- Makefile.inc2.in.orig	2014-08-06 14:43:42.000000000 +0000
+++ Makefile.inc2.in
@@ -47,7 +47,6 @@ INSTALL	=@DCCINSTALL@
 BININSTALL=$(INSTALL) -c $(SET_BINOWN) -m $(BINMODE)
 
 # create the DCC home directory before trying to install files
-beforeinstall:$(BINDIR) $(MANDIR)8 depend all
 $(BINDIR):
 	$(INSTALL) -d $(SET_BINOWN) -m 755 $(BINDIR)
 
